src/ui/articleview.cc

```diff
@@ -2199,11 +2199,17 @@ void ArticleView::highlightFTSResults()
     return;
   }
 
+  QString accuracy = "exactly";
+
+  if ( regString.contains( QRegularExpression( "[\\x4e00-\\x9fa5]+" ) ) ) {
```

Comment:
```suggestion
  if ( regString.front().script()==QChar::Script_Han ) {
```

Alternatively, there is QChar's script type